app/services/usps_exporter.rb

```diff
@@ -0,0 +1,54 @@
+class UspsExporter
+  OTP_MAX_VALID_DAYS = 30
+
+  def initialize(csv_file_path)
+    @csv_file_path = csv_file_path
+  end
+
+  def run
+    CSV.open(csv_file_path, 'wb', col_sep: '|') do |csv|
+      make_csv(csv)
+    end
+    clear_entries
+  end
+
+  private
+
+  attr_reader :csv_file_path, :entry_ids
+
+  def entries
```

Comment:
I'd probably list this below `make_csv` since that is what is using it.

I never know how to succinctly describe this guideline, so I searched through my Avdi Grimm archives for how he put it:

> dependencies in a given class should always point downward. That is, the first method in a class should depend on methods further down in the class definition, and those methods should depend on methods even further down, and so on.

This is both helpful for scanning (methods that are used first are listed first / easy to find) and for avoiding accidental circular dependencies.